code/src/components/ClearDoneButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+
+import { remember } from 'reducers/remember'
+import { Button } from 'lib/Button'
+
+export const ClearDone = () => {
+  const dispatch = useDispatch()
+
+  return (
+    <Button
+      background="salmon"
+      onClick={() => {
+        dispatch(remember.actions.clearDone())
+      }}>
```

Comment:
There is no need for this extra set of curly brackets, this is completely enough :

```
onClick={() => dispatch(remember.actions.clearDone())}>
```